app/components/Machines.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link as ChakraLink } from "@chakra-ui/next-js";
+import { Container, Heading } from "@chakra-ui/react";
+
+import "slick-carousel/slick/slick-theme.css";
+import "slick-carousel/slick/slick.css";
+
+import MachineCarousel from "./MachineCarousel";
+
+export default function Machines({ width }: { width: number }) {
+  const adjustedWidth = width * 0.95;
```

Comment:
On my Macbook air screen, the whole carousel doesn't fit and I have to use the horizontal scroll bar to see the right click arrow. My window zoom is 100%. When I change the multiplier from `0.95` to `0.9` I can see most of the slider, is this an issue on anyone else's screen?
<img width="1439" alt="Screenshot 2024-02-09 at 10 02 35 AM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/115492619/82930666-0c28-4197-9898-b631c4501903">

Reply:
There are definitely issues with it, so this is my first draft. I will look into why the percentage needs to be changed in this case.